sklearn/base.py

```diff
@@ -548,12 +560,18 @@ def score(self, X, y, sample_weight=None):
         return r2_score(y, y_pred, sample_weight=sample_weight)
 
     def _more_tags(self):
-        return {'requires_y': True}
+        return {'requires_y': True, 'estimator_type': 'regressor'}
 
 
 class ClusterMixin:
     """Mixin class for all cluster estimators in scikit-learn."""
-    _estimator_type = "clusterer"
+
+    # mypy error: Decorated property not supported
+    @deprecated("_estimator_type is deprecated in "  # type: ignore
+                "0.24 and will be removed in 0.26")
+    @property
+    def _estimator_type(self):
+        return "clusterer"
```

Comment:
it seems like the tag for clusterers doesn't exist?

Reply:
I forgot this one 😕.